resources/views/participants/form.blade.php

```diff
@@ -83,9 +83,22 @@
 	</div>
 
 	<div class="col-sm-2 form-group">
-		{!! Form::label('mag_gemaild', 'Mailings ontvangen:') !!}
-		{!! Form::select('mag_gemaild', [1 => 'Ja', 0 => 'Nee'], null, ['class' => 'form-control']) !!}
+		<label for="mag_gemaild">
+			Mailing ontvangen
+			<span class="glyphicon glyphicon-info-sign" aria-hidden="true" data-toggle="tooltip" title="Dit gaat alleen om nieuwsbrieven en kortingsacties. Bij deelname aan een kamp ontvangt u altijd mail.">
+		</label>
+		{!! Form::select('mag_gemaild', [0 => "Nee", 1=> "Ja"], 0, ['class' => 'form-control']) !!}
+		<i class="glyph">
 	</div>
+
+	<div class="col-sm-2 form-group">
+		<label for="post">
+			Kamp informatie 
```

Comment:
Ik zou denk ik gaan voor 'Informatie ontvangen via:' en dan opties 'post en email', 'alleen email'

Reply:
Oh en ik zou denk ik deze eerder plaatsen dan het 'mailings' vinkje (dus links ervan in het formulier, erboven in de show)

Reply:
Informatie ontvangen via wordt dan te groot
![image](https://user-images.githubusercontent.com/2979104/73007672-38ffb300-3e0d-11ea-9119-719be18fce02.png)
Misschien mailing ontvangen naar beneden als vinkje?